Snakefile

```diff
@@ -24,7 +24,15 @@ rule prepare_example:
     shell:
         "cp -r examples/{wildcards.scenario}/preprocessed results/{wildcards.scenario}/preprocessed"
 
-
+rule prepare_conv_pp:
+    input:
+        opsd="raw/conventional_power_plants_DE.csv",
```

Comment:
Strange indentation. Still some tabs?

Reply:
Should be right like that, everything works on my computer and I can count 4 spaces as tab.

Reply:
Sorry for the intrusion, but there are still some tabs in this file and trailing whitespace: https://github.com/rl-institut/oemof-B3/blob/579aba1fa2f18a8cd2a0aff1fd839630972cba4c/Snakefile

Reply:
Hi @b-r-u Thanks for giving us a visit :) Much appreciated! Can you take care of that, @juliusmeier?

Reply:
Thanks for the review! Yes, I take care of it.